reviews.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const db = require("./database_reviews/database.js");
+
+router.get("/reviews", async (req, res) => {
+  const data = await db.query("SELECT * FROM reviews");
+  return res.json(data);
+});
+
+module.exports = router;
```

Comment:
Are you using this code at all? I might be wrong, but from what I can tell you're not actually importing this file anywhere, so I'm not sure if it's serving any purpose.